app/perftools/management/commands/create_jtbd_cache.py

```diff
@@ -61,7 +61,7 @@ def create_jtbd_earn_cache():
 
     featured_grant = None
 
-    if not settings.DEBUG:
+    if settings.ENV in ['prod']:
```

Comment:
`WalletConnect` will only have this `grant_id` on production - I'd suggest that we make this check more explicit:

```suggestion
    if settings.ENV == 'prod':
```